index.html

```diff
@@ -1235,53 +1235,36 @@
   uses</a> for ongoing discussion and iteration on this principle.
 </aside>
 
-In maintaining duties of [=duty of protection|protection=], [=duty of discretion|discretion=] and [=duty of loyalty|loyalty=], user
-agents should be conservative when sharing data that isn't needed to satisfy a user's immediate goals.
+<div class="practice">
+<span class="practicelab">In maintaining duties of [=duty of protection|protection=], [=duty of discretion|discretion=] and [=duty of loyalty|loyalty=], user
+agents should take care that, when sharing data that isn't needed to satisfy a user's immediate goals, such information sharing
+aligns with the user's wishes and interests.</span>
```

Comment:
I'm not sure making this into a principle will be very actionable, but no objection.

There are no other references or definitions of user wishes, so maybe we could stick to interests or aligned with goals, unless we have an additional distinct meaning.

Reply:
"intents and interests" seems promising for the group